### ISSUE TITLE
subdivx: after latest patch not detecting "release" properly

### ISSUE THREAD

**Issue by zandadoum** [morpheus65535/bazarr#896]
Hello,

after I reported this:
https://github.com/morpheus65535/bazarr/issues/889
and it got solved, I inmediately installed latest "linuxserver/bazarr:development"
and while the previous problem was solved, I found a new one:

in the provider subdivx, apparenlty uploaders like to tell their lifestory in the comments (/sigh) and bazarr is picking that up as if it was the "release" info

example picture: https://imgur.com/EadEddh

in fact, bazarr is not even picking up the proper TV SHOW (i am looking for "x-men animated series" and the result is from "wolverine and the x-men"), yet showing a 91.94% match

i also checked a movie and as you can see in this link: https://www.subdivx.com/X6XNDIyNTc2X-stripes-1981.html

the field that is taken by bazarr as "release", on the providers webpage is more like a "comments" field and does rarely ever have a proper format

ERGO: i suggest ditching that field alltogether and just use that part at the top of the provider that sais "Subtitulo de Wolverine & the X-Men S01E01" which would be the proper tv show name and episode number and works for movies aswell

**Comment by morpheus65535** [comment]
@josdion something broken after latest fix?

**Comment by josdion** [comment]
Looks like subdivx return subtitle description as release_info. It may not have been so obvious before, since the release information was hidden behind the drop-down button. But after the last commit, it's always displayed because of this code:
`description = description.replace(",", " ").lower()`

**Comment by morpheus65535** [comment]
Something you can do about it? :-)

**Comment by josdion** [comment]
Of course I can help :)

- about release info
Didn't find on the site any information about release info except in the description. Not sure if it's a good idea to set the name of tv show or movie as a release info because they will be the same. I can probably come up with an idea where the description is shown as a pop-up after clicking on a button. 

- returning wrong results like wolverine and the x-men:
subdivx is searching for alternative titles too. I checked in sonarr and alternative title of X-Men: The Animated Series is just "X-Men". So subdivx will return all TV episodes that contain x-men in their names. Therefore, I will ask if it is necessary to look in alternative titles as well or I can remove it? Furthermore get_matches() expect that tv show name and movie names are already matched in search query.

I found one additional problem - the subtitles downloaded from the site are archive files that may contain more than one subtitle file - for different releases for example, but the provider will return only the first one. I am not sure if this is important because I have not used this site, but I have seen such file. [https://www.subdivx.com/X6XNTkxMDMxX-prodigal-son-s01e18.html](url)
